client/src/components/Dashboard/__tests__/Dashboard.test.jsx

```diff
@@ -0,0 +1,66 @@
+import React from "react";
+import { mount } from "enzyme";
+import { findByTestAttr } from "../../../test/testUtils";
+import Dashboard from "../Dashboard";
+import configureStore from "redux-mock-store";
+
+const mockStore = configureStore();
+const initialState = {
+  auth: {
+    isAuthenticated: true,
+    loading: false,
+    user: {
+      exp: 1589933753,
+      iat: 1558376827,
+      id: "5c6313a4c318bb62298b23d4",
+      name: "Jennifer"
+    }
+  }
+};
+
+const setup = (setupState = {}) => {
+  const store = mockStore(setupState);
+  return mount(<Dashboard store={store} />);
+};
+
+describe("Dashboard rendering", () => {
```

Comment:
Don't check in failing tests. I'd suggest not checking in this file until you've written the tests and they are passing.